avm/res/network/vpn-gateway/tests/e2e/waf-aligned/main.test.bicep

```
@@ -0,0 +1,106 @@
+targetScope = 'subscription'
+
+metadata name = 'WAF-aligned'
+metadata description = 'This instance deploys the module in alignment with the best-practices of the Azure Well-Architected Framework.'
+
+// ========== //
+// Parameters //
+// ========== //
+
+@description('Optional. The name of the resource group to deploy for testing purposes.')
+@maxLength(90)
+param resourceGroupName string = 'dep-${namePrefix}-network.vpngateways-${serviceShort}-rg'
+
+@description('Optional. The location to deploy resources to.')
+param location string = deployment().location
+
+@description('Optional. A short identifier for the kind of deployment. Should be kept short to not run into resource-name length-constraints.')
+param serviceShort string = 'vpngwaf'
+
+@description('Optional. A token to inject into the name of each resource.')
+param namePrefix string = '#_namePrefix_#'
+
+#disable-next-line no-hardcoded-location // Just a value to avoid ongoing capaity challenges
```

Small typo here and other similar occurrences
```suggestion
#disable-next-line no-hardcoded-location // Just a value to avoid ongoing capacity challenges
```